Added Japanese translation.

argentum-2503 opened the pull request:
Added ja_jp.json

argentum-2503 committed:
Added ja_jp.json

Anpan715 commented on common/src/main/resources/assets/refurbished_furniture/lang/ja_jp.json:
```diff
@@ -0,0 +1,650 @@
+    "block.refurbished_furniture.crimson_lattice_fence_gate": "真紅のラティスフェンスゲート",
+    "block.refurbished_furniture.warped_lattice_fence_gate": "歪んだラティスフェンスゲート",
+    "block.refurbished_furniture.mangrove_lattice_fence_gate": "マングロー のラティスフェンスゲート",
+    "block.refurbished_furniture.cherry_lattice_fence_gate": "サクラのラティスフェンスゲート",
```
「ラティスフェンス」よりも「格子付きフェンス」のほうが和訳らしいかなと思います。
あと465行に脱字があります。

argentum-2503 replied:
465行目の脱字は修正しました。
「ラティスフェンス」に関しては、該当ブロックが外構(Outdoors)カテゴリに分類されていることやグラフィックから、建築用語およびガーデニング用品におけるラティスフェンスと解釈しました。そのため先述の通りの和訳としています。

Anpan715 commented on common/src/main/resources/assets/refurbished_furniture/lang/ja_jp.json:
```diff
@@ -0,0 +1,650 @@
+    "computer_program.refurbished_furniture.paddle_ball.main_menu": "メニューへ戻る",
+    "computer_program.refurbished_furniture.paddle_ball.you": "あなた",
+    "computer_program.refurbished_furniture.paddle_ball.win_game": "勝ち！",
+    "computer_program.refurbished_furniture.paddle_ball.lose_game": "負け...",
```
ここは翻訳せずに 「You Win :)」 「You Lose :(」のままの方がレトロゲームっぽいかなと。

argentum-2503 replied:
「You Win :)」 「You Lose :(」に変更しました。

Anpan715 reviewed (approved):
いくつかの個人的な提案を挙げてみました。
正直個人的な意見のため無視してもらっても構いません。
あと265行に脱字もあります。

argentum-2503 committed:
Update ja_jp.json

Corrected omission in line 465.
Changed wording in lines 621 and 622.
465行目の脱字を修正しました。
621行目および622行目の表現を変更しました。
